parsers/AR.py

```diff
@@ -18,8 +18,23 @@
 CAMMESA_DEMANDA_ENDPOINT = (
     "https://api.cammesa.com/demanda-svc/generacion/ObtieneGeneracioEnergiaPorRegion/"
 )
+CAMMESA_EXCHANGE_ENDPOINT = (
+    "https://api.cammesa.com/demanda-svc/demanda/IntercambioCorredoresGeo/"
+)
 CAMMESA_RENEWABLES_ENDPOINT = "https://cdsrenovables.cammesa.com/exhisto/RenovablesService/GetChartTotalTRDataSource/"
 
+SUPPORTED_EXCHANGES = {
+    "AR->CL-SEN": "ARG-CHI",
+    "AR->PY": "ARG-PAR",
+    "AR->UY": "ARG-URU",
+}
```

Comment:
I noticed the endpoint support a lot more exchanges than just these. Could you add those as well?
They will be needed in #4894.

Reply:
I've added them to these dicts here. Should I add the exchange config files in this PR or the #4894?

Reply:
I would add the configs in the other PR as adding it in here would produce a lot of errors if when this is merged with exchanges that don't have the corresponding zones.